### ISSUE TITLE
New DataTable added virtualized list props

### ISSUE THREAD

**Issue by joshxg** [openmsupply/mobile#1140]
## Is your feature request related to a problem? Please describe.

Currently the use of `VirtualizedList` is very bare bones and standard. This could be improved to account for our particular use cases.

## Describe the solution you'd like

'Playing' with props of `VirtualizedList` in order to achieve the best feel and performance for our particular uses.

The props in particular that could be of interest:

- `getItemLayout`
- `maxToRenderPerBatach`
- `updateCellsBatchingPeriod`
- `initialNumToRender`

Just adding a basic `getItemLayout`

```
getItemLayout={(dataTwo, index) => ({
          length: dataTwo.length - 1,
          offset: dataTwo.length - 1 * index,
          index,
        })}
```

Seemed to decrease the layout time for sorting by about 1/2.

 Due to the size of our rows and the limited real estate being on a tablet, it would seem having smaller batches updated could also increase performance significantly while still having a pleasant UX

## Describe alternatives you've considered

Leaving it as is!

## Additional context
Not hugely high priority.. could be done anytime

Could also take a quick look at having some sort of loading indicator show to see if it's very simple

**Comment by Chris-Petty** [comment]
Yea worth doing some and adding as default props on `DataTable`

**Comment by joshxg** [comment]
Closed by #1043
